modules/bmaas-setting-up-a-dedicated-namespace.adoc

```diff
@@ -0,0 +1,141 @@
+// This module is included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bmaas-setting-up-a-dedicated-namespace_{context}"]
+= Setting up a dedicated namespace
+
+To prevent accidental interference between BMaaS workloads and the {product-title} infrastructure, it is strongly recommended to perform the following steps:
+
+* Configure the external BMaaS hosts in a dedicated namespace separate from the {product-title} cluster hosts.
+* Create a dedicated namespace for BMaaS.
+* Set up a user with minimal privileges. 
+* Apply the necessary Role-Based Access Control (RBAC) roles.
+
+.Procedure
+
+. Create a dedicated namespace for BMaaS hosts by running the following command:
++
+[source,terminal]
+----
+$ oc new-project bmaas
```

Comment:
Creating a namespace with this command also makes  it the current namespace. This step can be moved to the step after user secret and oauth configuration is updated.
`bmaas` namespace and `bmadmin` are example names. Both can have any name. Should we mention that?